audformat/core/database.py

```diff
@@ -1181,8 +1169,6 @@ def _set_attachment(
     ) -> Attachment:
         attachment._db = self
         attachment._id = attachment_id
-        if self.root is not None:
-            attachment._check_path(self.root)
         for other_id in list(self.attachments):
             attachment._check_overlap(self.attachments[other_id])
```

Comment:
This can raise a RuntimeError, which we do not mention in the docstring yet.

Reply:
I created https://github.com/audeering/audformat/issues/357 and will handle it in another pull request.